wpengine/faustjs
# MERL-1712: custom toolbar v2

## Description
Custom Toolbar example

## Testing
1. Setup a faust project using the custom-toolbar example
2. Login using the button on the main frontpage template to authenticate
3. Toolbar should display with the custom elements showing from CustomToolbar.js in the /plugins folder.

## Screenshots
